esp/src/src-react/hooks/grid.tsx

```diff
@@ -214,6 +215,7 @@ function useFluentStoreGrid({
             attribute: sorted ? column.key : "",
             descending: sorted ? isSortedDescending : false
         });
+        updateParam("sortBy", sorted ? (isSortedDescending ? "-" : "") + column.key : undefined);
```

Comment:
Move this logic into a `updateSort` function in the util/history, then all the logic for the ascending and descending will be encapsulated into the one file.